helm/odigos/templates/odiglet/clusterrole.yaml

```diff
@@ -34,37 +33,11 @@ rules:
   - apiGroups:
       - apps
     resources:
-      - daemonsets/finalizers
-      - deployments/finalizers
-      - statefulsets/finalizers
-    verbs:
-      - update
-  - apiGroups:
-      - apps
-    resources:
-      - daemonsets/status
       - deployments/status
+      - daemonsets/status
       - statefulsets/status
     verbs:
       - get
-  - apiGroups:
-      - odigos.io
-    resources:
-      - collectorsgroups
-      - collectorsgroups/status
```

Comment:
made collectorsgroup role instead of clusterrole. we should remove them and record the signals in instrumentationconfig (future PR)